Furball.Engine/Engine/Graphics/Drawables/UiElements/UiButtonDrawable.cs

```diff
@@ -0,0 +1,52 @@
+using Furball.Engine.Engine.Graphics.Drawables.Managers;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using MonoGame.Extended;
+
+namespace Furball.Engine.Engine.Graphics.Drawables.UiElements {
+    public class UiButtonDrawable : ManagedDrawable {
+        private string _text;
+        public string Text => this._text;
+        public TextDrawable TextDrawable;
+
+        private float _margin;
+        public float Margin => this._margin;
+
+        public Color OutlineColor = Color.Black;
```

Comment:
kinda very smol nitpick but would it be hard to add a outline thickness variable?

Reply:
Yup, should be easy to do, ill do that rq

Reply:
Resolved in 1899e7569aeb369ebb016ce0957067b79b872c20